.idea/codeStyles/codeStyleConfig.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectCodeStyleConfiguration">
```

Comment:
The only drawback with this approach
These don't kick in if someone uses an editor which is not IntelliJ

But the good part is that it will get formatted anyway with the precommit hook

Reply:
I guess it's on the eclipse user to figure out how to configure google java format in his IDE, for example.

We only "support" Intellij
https://docs.open-metadata.org/open-source-community/developer/build-code-run-tests#run-instance-through-intellij-idea